examples/quantizing_moe/deepseekv2_5_example.py

```diff
@@ -12,18 +12,17 @@
 # previous version or upgrading to a version where this bug is fixed
 
 # select a Mixture of Experts model for quantization
-MODEL_ID = "deepseek-ai/DeepSeek-Coder-V2-Lite-Instruct"
+MODEL_ID = "deepseek-ai/DeepSeek-V2.5"
 
 model = AutoModelForCausalLM.from_pretrained(
     MODEL_ID, torch_dtype=torch.bfloat16, trust_remote_code=True
 )
```

Comment:
Don't we need `device_map="auto"` to make this huge model fit?. I think it would be nice to still keep around a small moe example